controllers/component_controller.go

```diff
@@ -485,7 +498,10 @@ func (r *ComponentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		// If the model already exists, see if fields have been updated
 		log.Info(fmt.Sprintf("Checking if the Component has been updated %v", req.NamespacedName))
 
-		// Parse the Component Devfile
+		// Parse the Component CR Devfile
+		// Not necessary to pass in a Token or DevfileUtils client to the parser here since the devfileBytes has:
```

Comment:
same comment as above

Reply:
sniff tested on a live cluster after removing these tokens from the invocation and it worked well..